lib/install/planbuilder.go

```diff
@@ -525,10 +550,27 @@ func (i *Installer) GetPlanBuilder(cluster ops.Site, op ops.SiteOperation) (*Pla
 			UID:  strconv.Itoa(i.Config.ServiceUser.UID),
 			GID:  strconv.Itoa(i.Config.ServiceUser.GID),
 		},
-		DNSConfig: cluster.DNSConfig,
+		DNSConfig:               cluster.DNSConfig,
+		InstallerTrustedCluster: trustedCluster,
 	}, nil
 }
 
+// getInstallerTrustedCluster returns trusted cluster representing installer process
+func (i *Installer) getInstallerTrustedCluster() (storage.TrustedCluster, error) {
+	seedConfig := i.Process.Config().OpsCenter.SeedConfig
+	if seedConfig == nil {
+		return nil, trace.NotFound("process config is missing seed config: %#v",
```

Comment:
Yeah, these are tricky because it would effectively mean a programming error since seed config always contains trusted clusters for the installer process. I've updated the wording in attempt to make it clearer.